### PR TITLE
Newdemo

### PR DESCRIPTION
**Pull request by wty92911**
# add newdemo  
- 搜音乐返回对应url
- 修改了settings.py使得支持跨域访问
- 设置api url为服务器地址
- build dist

### CODE REVIEW

File: WebMusic/frontend/src/components/HelloWorld.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<template>
+  <div class="greetings">
+    <h1 class="green">{{ msg }}</h1>
+    <h3>
+      <el-row display="margin-top:10px">
+      <el-input v-model = "str" placeholder="请输入歌曲名"></el-input>
+      <el-button type="primary" @click="find">查询</el-button>
+    </el-row>
+    <el-row>
+<el-table :data="urllist" style="width: 100%" >
```
Comment (SueJane) [inline WebMusic/frontend/src/components/HelloWorld.vue:32]: fix the indent please

File: WebMusic/frontend/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import TheWelcome from './components/TheWelcome.vue'
+</script>
+
+<template>
+  <header>
+    <img alt="Vue logo" class="logo" src="./assets/logo.svg" />
+    <HelloWorld msg="Find Music" />
+    
+  </header>
+
```
Comment (SueJane) [inline WebMusic/frontend/src/App.vue:12]: remove all of the unused blank lines

File: WebMusic/frontend/src/components/HelloWorld.vue (hunk 1 above)
Comment (wty92911) [reply WebMusic/frontend/src/components/HelloWorld.vue:32]: 1

File: WebMusic/frontend/src/App.vue (hunk 2 above)
Comment (wty92911) [reply WebMusic/frontend/src/App.vue:12]: 1

**Review by SueJane — APPROVED** [review]

**Pull request merged**